auth-groups.json

```diff
@@ -20,6 +21,7 @@
     "Admission and Pupil Benefits": "education-evidence-production",
     "Leasehold and Right to Buy": "Leasehold and Right to Buy",
     "Housing Management": "DES Housing Management",
+    "Hackney Parking Services": "Hackney Parking Services - Evidence Drop Box",
```

Comment:
We shouldn't give them access to production yet. We will do that once they confirmed things are ok in staging.

Reply:
Addressed in [aae11f0](https://github.com/LBHackney-IT/document-evidence-store-frontend/pull/293/commits/aae11f03020c1dceeec7f7cf1965932a3644521d)